src/main/python/code_style/Config.py

```diff
@@ -0,0 +1,6 @@
+class Config:
+    def __init__(self, args: dict):
```

Comment:
```python
def __init__(self, source_root: str, formula: str, disable_single_character_word: str, variable_whitelist: str, **kwargs):
    ...
```

and you can pass the arguments like this
`Config(**args)`

Reply:
Seems not working :( 
`TypeError: Config.Config() argument after ** must be a mapping, not Namespace`

Reply:
@edisonhello 

It means the `args` must be a **dict-like** thing,  try `Config(**args.__dict__)`

If this still doesn't work, just pass each of them explicitly.

```python 
def __init__(self, source_root: str, formula: str, disable_single_character_word: str, variable_whitelist: str):
    self.source_root = source_root
    self.formula = formula 
   ... and so on 
```